Sources/MapboxMaps/Foundation/Camera/BasicCameraAnimator.swift

```diff
@@ -38,11 +38,8 @@ public class BasicCameraAnimator: NSObject, CameraAnimator, CameraAnimatorInterf
         }
     }
 
-    /// A timer used to delay the start of an animation
-    private var delayedAnimationTimer: Timer?
-
     /// The state from of the animator.
-    public var state: UIViewAnimatingState { propertyAnimator.state }
+    public var state: UIViewAnimatingState { return propertyAnimator.state }
```

Comment:
To make the diff even smaller…
```suggestion
    public var state: UIViewAnimatingState { propertyAnimator.state }
```